.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: php
+
+php:
+    - 5.4
+    - 5.5
+    - 5.6
+
+# test only master (+ Pull requests)
+branches:
+    only:
+        - master
+
+before_script:
+    - composer selfupdate
+    - composer install --prefer-source
```

Comment:
to speed up things `--prefer-dist` is probably better (unless I miss something ?)